src/main/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/domene/PersonResultat.kt

```diff
@@ -128,6 +128,28 @@ class PersonResultat(
         return nyttPersonResultat
     }
 
+    fun tilKopiForNyVilkårsvurdering(
+        nyVilkårsvurdering: Vilkårsvurdering,
+    ): PersonResultat {
+        val nyttPersonResultat = PersonResultat(
+            vilkårsvurdering = nyVilkårsvurdering,
+            aktør = aktør,
+            andreVurderinger = mutableSetOf(), // Vi kopierer ikke over andreVurderinger da den aldri skal være med i ny behandling
+        )
+
+        val nyeVilkårResultater = vilkårResultater
+            .map {
+                it.tilKopiForNyttPersonResultat(
+                    nyttPersonResultat = nyttPersonResultat,
+                )
+            }
+            .toSortedSet(VilkårResultatComparator)
+
+        nyttPersonResultat.setSortedVilkårResultater(nyeVilkårResultater)
```

Comment:
Her tolker jeg det som at `setSortedVilkårResultater` sorterer, så burde ikke trenge sorteringen

```suggestion
            }.toSet()

        nyttPersonResultat.setSortedVilkårResultater(nyeVilkårResultater)
```